smart-mon.spec

```diff
@@ -40,9 +42,12 @@ binary and corresponding service.
 %install
 install -m 755 -d %{buildroot}%{bin_dir}/
 install -m 755 -d %{buildroot}%{script_dir}/
+install -m 755 -d %{buildroot}%{service_dir}/
 install -m 755 -d %{buildroot}%{etc_ceph_dir}/
+install -m 755 -d %{buildroot}%{node_exporter_dir}/
 install -m 755 smartmon.sh %{buildroot}%{bin_dir}
 install -m 755 script.sh %{buildroot}%{script_dir}
+install -m 644 smart.service  %{buildroot}%{service_dir}
 install -m 755 node-exporter.yml %{buildroot}%{etc_ceph_dir}
 
```

Comment:
Should add systemd macros for the service:

```
# helps when installing a program whose unit files makes use of a feature only available in a newer systemd version
# If the program is installed on its own, it will have to make do with the available features
# If a newer systemd package is planned to be installed in the same transaction as the program,
# it can be beneficial to have systemd installed first, so that the features have become available by the time program is installed and restarted
%{?systemd_ordering}
```
and
```
%pre
%service_add_pre smart.service

%post
%service_add_post smart.service

%preun
%service_del_preun smart.service

%postun
%service_del_postun smart.service
```